### PR TITLE
Class command and modified role property to include class

### PR DESCRIPTION
**Pull request by tommymurphy**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -152,6 +149,26 @@ client.on('message', message => {
...
+			return;
+		}
+
+		if(!MAPLE_STORY_CLASSES.includes(req.toLowerCase())){
+			message.channel.send(`Zakum tried his best but does not think that ${req} exists in MapleStory M.`)
+			return;
+		}
+
+		const classList = roster.filter(member => member.role.toLowerCase() === req.toLowerCase()).map(member => member.name)
+		classList && classList.length ? message.channel.send(`${classList.toString()} (${classList.length})`) : message.channel.send(`Zakum could not locate any guild members that play ${req}.`)
```
Comment (driedyam) [inline bot.js]: classList.toString() => classList.join(', ') //c'mon man, you're better than this. lol

File: bot.js [hunk 2]
```diff
@@ -152,6 +149,26 @@ client.on('message', message => {
 		});
 	}
 
+	if(message.content.substring(0,6) === `${prefix}class`){
+		const args = message.content
+		const req = args.split(" ").length === 3 ? `${args.split(" ")[1]}${args.split(" ")[2]}`:args.split(" ")[1]
```
Comment (driedyam) [inline bot.js]: `const req = args.slice(1).join('')`

File: bot.js (hunk 1 above)
Comment (tommymurphy) [reply bot.js]: why you gotta roast me like that :(

**Comment by driedyam** [comment]
just messing around. you good you good.

File: bot.js (hunk 2 above)
Comment (tommymurphy) [reply bot.js]: This does not work

File: bot.js (hunk 1 above)
Comment (driedyam) [reply bot.js]: sorry sorry

File: bot.js (hunk 2 above)
Comment (driedyam) [reply bot.js]: whoops, left a part out

Comment (driedyam) [reply bot.js]: still need to split it, but just discard the first index, and join everything after

**Comment by driedyam** [comment]
Looks good! Thanks for doing the needful so kindly.

**Pull request merged**